templates/partners/silicon.html

```diff
@@ -115,8 +115,9 @@ <h2>Reduce development friction and time to market</h2>
         <p>Pro-active pre GA(general availability) work on next-generation software and silicon hardware through optimised Ubuntu offering ensures silicon features reach both internal and external users in the fastest way possible. Strong Ubuntu community engagement ensures constant evolution of hardware and software features.</p>
         <p>Explore how the joint solutions offered by Canonical and our silicon partners accelerate the customers' project.</p>
         <ul class="p-list--divided">
+          <li class="p-list__item is-ticked"><a href="https://ubuntu.com/engage/partners-benefits-of-working-with-canonical-in-iot">Read the whitepaper: Scale, accelerate, and reduce time-to-market throughout IoT development</a></li>
```

Comment:
I don't see these links in the provided copy doc: https://docs.google.com/document/d/1nyiz4I1GqBiXe_SKpDzho3yv20LKyYpyhHh0fvvUrOI/edit. It's probably because it in the wrong folder and so I do not have permission to see the suggestions. Can you as the PM to move it to the correct place please?